Src/Sarif.PatternMatcher.Sdk/Fingerprint.cs

```diff
@@ -227,6 +226,29 @@ public string GetValidationFingerprintHashText(bool jsonFormat = false)
             return ComputeHash(validationFingerprint);
         }
 
+        /// <summary>
+        /// Returns a fingerprint comprised of the minimal information that
+        /// uniquely identifies a credential at a specific point-in-time.
+        /// </summary>
+        /// <returns>The minimal fingerprint that uniquely identifies a credential.</returns>
+        public string GetUniqueSecretFingerprint()
+        {
+            return new Fingerprint()
+            {
+                Secret = this.Secret,
+            }.ToString();
+        }
+
+        /// <summary>
+        /// Returns the hash of the minimal fingerprint that uniquely
+        /// identifies a credential at a specific point-in-time.
+        /// </summary>
+        /// <returns>The minimal fingerprint that uniquely identifies a credential.</returns>
+        public string GetUniqueSecretHash()
```

Comment:
@michaelcfanning Is this hash guaranteed to be equal to the hash generated by the legacy SPAM tool?